sda/host.go

```diff
@@ -92,14 +93,18 @@ func (h *Host) listen(wait bool) {
 		h.registerConnection(c)
 		h.handleConn(c)
 	}
-	go func() {
```

Comment:
`Listen` is non-blocking now, as we need to know when the port-number is available.